packages/api/src/task/scheduler.ts

```diff
@@ -298,5 +299,23 @@ export default class TaskScheduler {
         },
       },
     });
+    if (!updates.status || updates.status === asset.status) {
+      return;
+    }
+    if (asset.status.phase == "ready" || asset.status.phase == "failed") {
```

Comment:
**humanlint:** Always use triple equals `===` in JavaScript unless you know you need double equals `==`. (oh javascript... 😆)

```suggestion
    if (asset.status.phase === "ready" || asset.status.phase === "failed") {
```